velox/exec/fuzzer/PrestoQueryRunner.cpp

```diff
@@ -291,7 +306,11 @@ std::optional<std::string> PrestoQueryRunner::toSql(
     for (auto i = 0; i < aggregates.size(); ++i) {
       appendComma(i, sql);
       const auto& aggregate = aggregates[i];
-      sql << toCallSql(aggregate.call);
+      VELOX_CHECK(
+          !aggregate.distinct,
+          "Presto Query Runner doesnt support distinct aggregates");
```

Comment:
typo: doesnt -> doesn't or does not

Would be nice to remote this limitation in a follow-up. Should be straightforward.

Reply:
Will remove in follow up.